Package.swift

```diff
@@ -15,7 +15,7 @@ let package = Package(
         .package(
             name: "MuxCore",
             url: "https://github.com/muxinc/stats-sdk-objc.git",
-            from: "3.2.0"
+            .branch("nidhik/codesign")
```

Comment:
this was for testing purposes, before merging it should change back to `from: "3.2.1"`